pkg/tests/tasks/security/authorization/deny_test.go

```diff
@@ -16,149 +16,177 @@ package authorization
 
 import (
 	"fmt"
-	"strings"
 	"testing"
-	"time"
 
-	"github.com/maistra/maistra-test-tool/pkg/examples"
-	"github.com/maistra/maistra-test-tool/pkg/util"
-	"github.com/maistra/maistra-test-tool/pkg/util/log"
+	"github.com/maistra/maistra-test-tool/pkg/app"
+	"github.com/maistra/maistra-test-tool/pkg/util/check/assert"
+	"github.com/maistra/maistra-test-tool/pkg/util/hack"
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/pod"
+	"github.com/maistra/maistra-test-tool/pkg/util/retry"
 	"github.com/maistra/maistra-test-tool/pkg/util/test"
 )
 
-func cleanupAuthorDeny() {
-	log.Log.Info("Cleanup")
-	util.KubeDeleteContents("foo", AllowPathIPPolicy)
-	util.KubeDeleteContents("foo", DenyHeaderNotAdminPolicy)
-	util.KubeDeleteContents("foo", DenyGETPolicy)
-	time.Sleep(time.Duration(40) * time.Second)
-	sleep := examples.Sleep{Namespace: "foo"}
-	httpbin := examples.Httpbin{Namespace: "foo"}
-	sleep.Uninstall()
-	httpbin.Uninstall()
-	time.Sleep(time.Duration(20) * time.Second)
+func TestAuthorizationDenyAllow(t *testing.T) {
+	test.NewTest(t).Id("T23").Groups(test.Full, test.InterOp).Run(func(t test.TestHelper) {
+		hack.DisableLogrusForThisTest(t)
```

Comment:
Is this a temporary method for disabling logrus ? Do we need to include this line in other test cases refactoring ?

Reply:
Yes, temporary until we refactor all the tests. It's not really necessary to call this function, but it prevents the low-level util functions from logging their own stuff, leaving just the high-level log statement from the test and the new framework. It makes the log output a bit easier to read, but doesn't show all the details.